### DISCUSSION TITLE
Limit reconstruction to arbitrary 2D plane

### DISCUSSION THREAD

**Discussion by chrchr-github** [astra-toolbox/astra-toolbox#468] [Q&A]
Preface: I'm a CT novice, so please bear with me.

Is it possible to limit reconstruction to an arbitrary plane in 3D space (as opposed to the XY, XZ, or YZ planes)?
The idea is to avoid a fully 3D reconstruction to save computing time.
I have been looking at a few libraries including ASTRA but never saw this option. So am I missing something, or maybe does this idea  not make sense for fundamental reasons?

**Comment by askorikov** [comment]
Hi! You can achieve this by creating a volume geometry with 1 slice, and rotating  the source and detector appropriately to select the desired slice orientation. Unfortunately, Astra doesn't support rotating the volume geometry out-of-the-box, that's why you need to go this route. You can also look into Tomosipo library (a high level interface for Astra), particularly [this example](https://aahendriksen.gitlab.io/tomosipo/intro/lab_frame.html), which may make this easier.

**Reply by chrchr-github** [reply]
Thanks, I'll look into it.
